examples/generator-fullstack/src/schema/generated.ts

```diff
@@ -0,0 +1,79 @@
+
+export const schemaString = `
+
+
+type Note {
```

Comment:
Shoul not be commited. Let's clean example app completely. 
There is test case that tests migration that we should change instead.